frontend/src/components/pages/reassign-partitions/Step1.Partitions.tsx

```diff
@@ -323,9 +320,7 @@ function renderPartitionErrorsForTopic(_partitionsWithErrors: number) {
     >
         <span>
             <ZeroSizeWrapper justifyContent="center" alignItems="center" width="20px" height="18px">
-                <span style={{fontSize: '19px'}}>
-                    <WarningTwoTone twoToneColor="orange"/>
-                </span>
+                <MdOutlineWarningAmber color="orange" size={20} />
```

Comment:
Why is there different sizing for MdOutlineWarningAmber in various places?

Reply:
I think I'll answer both of your question in one comment. At first, I tried to keep everything unified, but the current implementation also did have various sizes on various places + MD icons are visually positioned differently than ant-design icons if we keep the same dimensions.

So I tried to adjust this to reduce/minify visual regressions, even though it was a bit handy to do. I think we can make it continuously more consistent over time (and we should). But that would require much more time now, and I think removing this dependency is not so business critical now.